### DISCUSSION TITLE
guidescript scalability

### DISCUSSION THREAD

**Discussion by aliabudiab** [Yawanseh/guidescript#12] [Q&A] [answered]
Wow! This package is truly impressive! I was wondering if it will be scalable enough to maintain all the features covered within the guidance package.

**Comment by Yawanseh** [comment, marked as answer]
That's a fantastic question! Indeed, the Guidance package is pretty comprehensive, and while our goal is to provide you with as many functionalities as possible, it's important to note that we're still laying the groundwork.

However, one of the most exciting parts of this project is its community-driven aspect. We highly value your input and encourage you to join us in this development journey. We believe that with your support and active participation, we can make significant strides towards fully integrating Guidance's features.

If you have a particular feature in mind that you'd like to see in our package, please don't hesitate to raise an issue. We promise to do our best to incorporate it. After all, our ultimate goal is to support the JS community.

Thank you for being a part of this exciting journey! 🙌

**Comment by Erudition** [comment]
I'm also excited by the potential!
Has development stalled?
